### DISCUSSION TITLE
OCPP commands via local API

### DISCUSSION THREAD

**Discussion by reulnl** [goecharger/go-eCharger-API-v2#196] [Ideas]
Would it be possible to add functionality to the local API to post OCPP commands?

It would be great if I could use e.g. the `RemoteStartTransaction` command and specify an `idTag` value via the local API to remotely start a charging transaction, that will be registered as a transaction on the OCPP server.

**Comment by erxbout** [comment]
Wouldn´t that allow injection of ocpp commands for example over the hotspot of the charger..
Not sure if that would make any sense if you have control over your own ocpp backend..?

Why not using it as intended and present the rfid card?
